todoman/cli.py

```diff
@@ -53,14 +53,25 @@ def _validate_list_param(ctx, param=None, name=None):
             raise click.BadParameter(
                 'You must set "default_list" or use -l.'.format(name)
             )
-    for l in ctx.db.lists():
-        if l.name == name:
-            return l
-    else:
-        list_names = [l.name for l in ctx.db.lists()]
-        raise click.BadParameter(
-            "{}. Available lists are: {}".format(name, ', '.join(list_names))
-        )
+    lists = {
+        list_.name: list_
+        for list_ in ctx.db.lists()
+    }
+    normalized_lists = {
+        list_.name.lower(): list_
+        for list_ in lists.values()
+    }
+    original_name = name
+    if len(lists) == len(normalized_lists):
```

Comment:
Can't we use normalized lists in more situations? Imagine the following:

* We have three lists: `a`, `A`, `b`
* The user selects the list `B`

In this situation not *all* list names are unique, but the one *we care about* is.

Reply:
Good catch, I hadn't really though about that. I guess you'd have to count how many lists normalize the the same as the input one, and make sure that's `== 1`.